src/layouts/EditHomepage.jsx

```diff
@@ -1171,14 +1173,36 @@ const EditHomepage = ({ match }) => {
   return (
     <>
       {itemPendingForDelete.id && (
```

Comment:
inline into the `isOpen` prop

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/513eedac744b2fe3c2814aff9b935982147c46d2